src/host/cli-run-plan.js

```diff
@@ -136,6 +137,28 @@ export const builder = (args = yargs) =>
         choices: ['request', 'skip'],
         hidden: true,
       },
+      'callback-url': {
+        describe: 'URL to POST test results to as they complete',
+        default: process.env.ARIA_APP_CALLBACK_URL,
+      },
+      'callback-header': {
+        describe: 'Header to send with callback request',
+        default: process.env.ARIA_APP_CALLBACK_HEADER,
+        // set to hidden if the callback header is set via environment (to not accidentally leak secret in a log)
+        hidden: Boolean(process.env.ARIA_APP_CALLBACK_HEADER),
```

Comment:
[This seems like a job for yargs' `.env` method.](https://github.com/yargs/yargs/blob/e517318cea0087b813f5de414b3cdec7b70efe33/docs/api.md#envprefix)

Reply:
[28e292c](https://github.com/w3c/aria-at-automation-harness/pull/20/commits/28e292cb2b5edd16fed0f8be37bf98a6dc46425c)